internal/internal.go

```diff
@@ -0,0 +1,171 @@
+package internal
+
+import (
+	"github.com/casbin/casbin/v3/errors"
+	"github.com/casbin/casbin/v3/model"
+	"github.com/casbin/casbin/v3/persist"
+	"github.com/casbin/casbin/v3/rbac"
+)
+
+// InternalController is the internal controller for model and adapter
+type InternalController interface {
```

Comment:
The name looks not clear, I think that we should rename InternalController to `PolicyManager`. 

@dovics @GopherJ @hsluoyz What do you think?

Reply:
We should return the affected data when adding/removing multiple policies.

Reply:
Why not call it `InternalEnforcer`?

Reply:
@hsluoyz At present, casbin supports updating policy via enforcer.AddPolicy/RemovePolicy or get model way. For get model way, after updating the policy, users need to call `adapter`, `watcher`, this is complex for users, we have to simplify the operation, so we extract a layer for other plug-ins/user to update the policy, this is also because of casbin-raft. when the `dispatcher` is enabled, the ManagerAPI provided by casbin will forward the request to casbin-raft when policy changes.